src/menu/maybe-child-menu.js

```diff
@@ -44,6 +44,7 @@ export default function MaybeChildMenu(props: PropsT) {
             mountNode={ctx.mountRef.current ? ctx.mountRef.current : undefined}
             onMouseEnterDelay={30}
             onMouseLeaveDelay={30}
+            onEsc={ctx.onEsc ? ctx.onEsc : undefined}
```

Comment:
Thanks for adding this, but at the moment I believe it's unmounting a bit too much on escape. This should call `resetParentMenu` from props. Calling that should close the top-most child menu and move selection back to the parent.

Reply:
Ah, didn't notice that method- seems to do the trick. I'm wondering if its okay to have it be the default (esc resets parent menu). I pushed up changes to review.